wagtail_purge/wagtail_hooks.py

```diff
@@ -1,7 +1,7 @@
 from django.urls import include, reverse, path
 from wagtail.admin.menu import AdminOnlyMenuItem
 
-from wagtail import hooks
+from wagtail import hooks, VERSION as WAGTAIL_VERSION
```

Comment:
```suggestion
from wagtail import hooks
```

since `WAGTAIL_VERSION` is not used anywhere

Reply:
Applied this too